pkg/placement/placement.go

```diff
@@ -124,10 +134,10 @@ func runtimeObjectToPlacement(obj runtime.Object) (*v1alpha1.Placement, error) {
 // read objects from all listers and enqueue the keys
 // this is useful for when a new placement is added
 // or a placement is updated
-func (c *Controller) requeueAll() error {
+func (c *Controller) requeueAllObjects() error {
```

Comment:
FYI: So maybe "Workload" would be better than "All" in this func's name? What about CRD objects?

Reply:
This and the above comment would change the existing flow, which I do not intend to change until we're set for disabling existing transport-logic and enabling the new one, which would either be the final phase of this PR or a new PR.